libs/infra-metrics/src/lib/dogStatsD.ts

```diff
@@ -7,4 +11,13 @@ export class DogStatsD extends StatsD {
       prefix: `islandis.${options.prefix}`.replace(':', '.'),
     })
   }
+
+  static timer() {
+    return process.hrtime.bigint()
+  }
+
+  static duration(start: bigint): number {
+    const diff = process.hrtime.bigint() - start
+    return Number(diff) / 1e6
+  }
```

Comment:
Is this the correct place for this? Or some `util`?

Reply:
Fancy hrtime bigint logic! Does StatsD support this precision? :)

This is a fine location but might change depending on my other suggestions.

Reply:
No, we turn it into regular ol' number before sending it off.
[Node 18 deprecates `process.hrtime()`](https://nodejs.org/docs/latest-v18.x/api/process.html#processhrtimetime)